src/Platform/Microsoft.Testing.Platform/Messages/ConsumingEnumerableConsumerDataProcessor.cs

```diff
@@ -10,17 +10,14 @@ namespace Microsoft.Testing.Platform.Messages;
 
 internal sealed class AsyncConsumerDataProcessor : IDisposable
```

Comment:
TODO: Hide the previous implementation under env variable, just in case something breaks.